frontend/app/forms/bank/form-bank.component.js

```diff
@@ -267,6 +269,66 @@ export default class FormBank extends React.Component {
             />
           )}
         />
+        <Route
+          path={`${this.props.match.url}/proof-of-completed-service`}
```

Comment:
We were just missing routes for these other forms? Why are these being added here?

Reply:
Result of a bad conflict resolution. These were duplicates.